src/actions/CardActions.ts

```diff
@@ -15,8 +16,8 @@ export const MAYBEBOARD = {
     REMOVE: "MAYBEBOARD_REMOVE",
 }
 
-export const addToDeckList = (cardInfo) => {
-    return (dispatch, getState) => {
+export const addToDeckList = (cardInfo: CardInfo) => {
```

Comment:
update CardInfo and avoid using any in this file

Reply:
I'm going to come back to this. Hard to find examples of how to handle dispatch with redux thunks